### PR TITLE
clear the lastAccessedTime for all tags when clearing history

### LINKED ISSUE #3616: bookmarked pages not deleted from history after restarting
**Describe the issue you encountered:** bookmarked pages are not deleted from about:history after restarting with the notification bar.

**Expected behavior:** 

- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 10 32bit

- Brave Version: 0.11.6 RC2

- Steps to reproduce:
    1. Create a backup
    2. Open a page and bookmark it
    3. Enable "Browser history" of Private data options on ``about:preferences#security``
    4. Disable/enable smooth scrolling to display the notification bar
    5. Click "Yes"
    6. Open ``about:history``

- Screenshot if needed:

- Any related issues:

### PR DESCRIPTION
**Pull request by bridiver**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

fixes #3616

### CODE REVIEW

File: app/sessionStore.js [hunk 1]
```diff
@@ -252,7 +252,12 @@ module.exports.cleanAppData = (data, isShutdown) => {
   if (data.sites) {
     const clearHistory = isShutdown && getSetting(settings.SHUTDOWN_CLEAR_HISTORY) === true
     if (clearHistory) {
+      // TODO - this should the history methods from siteUtils
```
Comment (bbondy) [inline app/sessionStore.js]: don't understand this todo, maybe missing a word `use`

File: app/sessionStore.js [hunk 2]
```diff
@@ -252,7 +252,12 @@ module.exports.cleanAppData = (data, isShutdown) => {
...
       data.sites = data.sites.filter((site) => site && site.tags && site.tags.length)
+      data.sites = data.sites.map((site) => {
+        delete site.lastAccessedTime
```
Comment (bbondy) [inline app/sessionStore.js]: is it save to delete it instead of setting to 0? /cc @bsclifton

**Comment by bsclifton** [comment]
Great catch @bridiver :smile:

File: app/sessionStore.js (hunk 2 above)
Comment (bsclifton) [inline app/sessionStore.js]: The clear modal and context menu in about:history set the value to null, but this works too.  Verified by cherry picking into my branch and manually testing :+1:

**Comment by bbondy** [comment]
++ for 0.11.7

File: app/sessionStore.js (hunk 1 above)
Comment (bridiver) [inline app/sessionStore.js]: it's hard to form complete sentences sometimes

**Pull request merged**